logical_scenarios/alks_scenario_4_1_1_free_driving_variation.xosc

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c) 2020-2024, Bayerische Motoren Werke Aktiengesellschaft (BMW AG) -->
 <OpenSCENARIO>
-  <FileHeader revMajor="1" revMinor="1" date="2021-07-06T10:00:00" description="Variation for ALKS Scenario 4.1_1 Free Driving" author="BMW AG">
+  <FileHeader revMajor="1" revMinor="3" date="2021-07-06T10:00:00" description="Variation for ALKS Scenario 4.1_1 Free Driving" author="BMW AG">
```

Comment:
BTW: What date ist this? Should this be updated to the date the scenario was updated?